dts/arm/nuvoton/npcx/npcx.dtsi

```diff
@@ -44,7 +44,7 @@
 		 * no_ks000-17 - PIN21.20.17.16.15.14.13.12.11.10.07.06.05.04.
 		 *                  82.83.03.B1
 		 */
-		pinmux = <>;
+		pinmux = < >;
```

Comment:
if `soc {};` style is used above, shouldn't this be `pinmux = <>;` ?

Reply:
Good point! I can add this rule to linter to make sure that `<>` and `[]` also have no spaces and on same line.